src/App.js

```diff
@@ -1,10 +1,16 @@
 import React from "react";
+
+import { mockPokemonData } from "./mock/pokeData";
+import PokeCard from "./components/PokeCard";
 import "./styles.css";
 
 export default function App() {
   return (
     <div className="App">
       <h1>Welcome to your second mini challenge!</h1>
+      <div className="pokemonsContainer">
+        <PokeCard pokemonData={mockPokemonData} />
```

Comment:
Since there are not much elements inside the mocked data, you could send them as properties for having the code more descriptive itself
```suggestion
     const { name, sprites, video } = mockPokemonData;
      <PokeCard sprites={sprites} name={name} video={video} />
```